.husky/pre-commit

```diff
@@ -1,6 +1,7 @@
 if [ -d "$(git rev-parse --git-path rebase-merge)" ] || [ -d "$(git rev-parse --git-path rebase-apply)" ]; then
-  echo "Rebase in progress. Skipping pre-commit hook.\n";
-  exit 0;
+  echo "Rebase in progress. Skipping pre-commit hook.\n"
+  exit 0
 fi
 
 lint-staged
+npm run check-types # Run tsc on the whole project, as running it only on staged files may result in import or type errors.
```

Comment:
Probably better to think of type checking as something different from "linting" and remove it from scripts with that name.

Reply:
https://github.com/Automattic/remote-data-blocks/pull/185/commits/7ce75c6689953bc0eb331798e4bd2bc6a7c39717